cli/utils/init.config.ts

```diff
@@ -10,21 +12,33 @@ export default function provideInitConfig(
   shell: typeof shelljs,
   filesystem: typeof fs,
   fortaKeystore: string,
-  configFilename: string
+  configFilename: string,
+  agentId: string,
+  getFortaConfig: GetFortaConfig
 ) {
   assertExists(shell, "shell");
   assertExists(filesystem, "filesystem");
   assertIsNonEmptyString(fortaKeystore, "fortaKeystore");
   assertIsNonEmptyString(configFilename, "configFilename");
 
   return async function initConfig() {
-    if (!filesystem.existsSync(join(fortaKeystore, configFilename))) {
+    const filePath = join(fortaKeystore, configFilename);
+    if (!filesystem.existsSync(filePath)) {
+      // Create file
       console.log(`Creating ${configFilename}...`);
       const copyConfigResult = shell.cp(
         join(__dirname, "..", "commands", "init", configFilename),
         fortaKeystore
       );
       assertShellResult(copyConfigResult, `Error creating ${configFilename}`);
+
+      const fortaCon = getFortaConfig();
```

Comment:
since the **project** forta.config.json will not exist and we'll be creating it here, we dont need to call `getFortaConfig`

Reply:
Removed